[oracle] Fix used bytes logic

niraj-elastic opened the pull request:
## Description
The field oracle.tablespace.space.used.bytes is calculated wrongly

Currently we are showing [total bytes of tablespace](https://github.com/elastic/integrations/blob/b3c56a72a35627e47ec0f6f73aa02822161ff4ae/packages/oracle/data_stream/tablespace/agent/stream/stream.yml.hbs#L45C17-L47C39) in integration which seems to be incorrect for the field description. This PR will update this query to do following calculation `space.used.bytes` = [`Total bytes`](https://github.com/elastic/integrations/blob/b3c56a72a35627e47ec0f6f73aa02822161ff4ae/packages/oracle/data_stream/tablespace/agent/stream/stream.yml.hbs#L46C25-L46C35) - [`Free Bytes`](https://github.com/elastic/integrations/blob/b3c56a72a35627e47ec0f6f73aa02822161ff4ae/packages/oracle/data_stream/tablespace/agent/stream/stream.yml.hbs#L41C18-L43C39)

Also fixed visualization related to total.bytes

In  visualization "Table space total size" There are two defrent queries are used for counting total bytes.
For NON-TEMP we are showing used+free which is correct.
![unnamed (5)](https://github.com/elastic/integrations/assets/124254029/52feb7de-4bc9-41e9-9dd5-3bddc21acb54)

But for TEPM we are directly showing total bytes. which is incorrect and shows total of all the tablespace.
![unnamed (6)](https://github.com/elastic/integrations/assets/124254029/e335fc55-30c7-4f75-8034-60ab9dc8c139)


## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally
- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/orecle) directory.
- Run the following command to run tests. elastic-package test

## Related issues

- Closes #9889 
<!-- Recommended
Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.

- Closes #123
- Relates #123
- Requires #123
- Supersedes #123
-->

elasticmachine commented:
### :rocket: Benchmarks report




To see the full report comment with `/test benchmark fullreport`
<!--COMMENT_GENERATED_WITH_ID_benchmark-report-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11799)
* Commit: 43450a3184b8b63df52197e1d7caecc943fc3e48

cc @niraj-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11799","number":11799,"commit":"43450a3184b8b63df52197e1d7caecc943fc3e48"}],"number":11799}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11954)
* Commit: 635807b0c0ccc6210d05ddc6587aa702acfb9cdf

### History
* :green_heart: [Build #11799](https://buildkite.com/elastic/integrations/builds/11799) succeeded 43450a3184b8b63df52197e1d7caecc943fc3e48

cc @niraj-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11954","number":11954,"commit":"635807b0c0ccc6210d05ddc6587aa702acfb9cdf"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11799","number":11799,"commit":"43450a3184b8b63df52197e1d7caecc943fc3e48"}],"number":11954}
buildkite-pr-comment-->

ishleenk17 commented:
Can you share the before and after of the dashboard? Are there changes other than label change in dashboard?

ishleenk17 commented on packages/oracle/validation.yml:
```diff
@@ -1,3 +0,0 @@
-errors:
-  exclude_checks:
-    - SVR00002
```
More details on this, please. 
How is this validation removal being handled in the PR?

niraj-elastic replied:
Since the datastream.data set filter is already added to oracle. we don't need to exclude this check. So we can remove this check file.

agithomas reviewed (approved):
LGTM!

milan-elastic reviewed (approved):
LGTM!

niraj-elastic commented:
> @niraj-elastic Can you share the before and after of the dashboard? Are there changes other than label change in dashboard?

@ishleenk17 The labels are added in this [PR](https://github.com/elastic/integrations/pull/9655) which is merged now. we have changed one visualization which has total bytes. I have updated the description with more details of change.

ishleenk17 commented:
> > @niraj-elastic Can you share the before and after of the dashboard? Are there changes other than label change in dashboard?
> 
> @ishleenk17 The labels are added in this [PR](https://github.com/elastic/integrations/pull/9655) which is merged now. we have changed one visualization which has total bytes. I have updated the description with more details of change.

Thanks for updating the description. It’s much more informative now.

elasticmachine commented:
Package oracle - 1.26.1 containing this change is available at https://epr.elastic.co/search?package=oracle
